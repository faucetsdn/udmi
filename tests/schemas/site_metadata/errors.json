{
  "version": "1.5.0",
  "tags": ["hq"],
  "location":{
      "address": "1600 Amphitheatre Parkway Mountain View, CA 94043",
      "lat": 	37.422,
      "long": 122.084
  },
  "parameters":{
    "area":{
      "net_occupied_area":{
        "value": 123,
        "unit": "badunit"
      },
      "extra":{
        "value": 123,
        "unit": "sqf"
      }
    }
  }
}