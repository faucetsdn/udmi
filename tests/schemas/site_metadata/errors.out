3 schema violations found
  /parameters/area/net_occupied_area/unit: instance failed to match exactly one schema (matched 0 out of 7)
  /parameters/area: object instance has properties which are not allowed by the schema: ["extra"]
  object has missing required properties (["timestamp"])
